src/navigation/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {
+    CardStyleInterpolators,
+    createStackNavigator,
+    StackNavigationOptions,
+} from '@react-navigation/stack';
+import Internal from '../screens/Internal';
+import Home from '../screens/Home';
+
+const {Navigator, Screen} = createStackNavigator();
+
+const animatedNavigation: StackNavigationOptions = {
+    gestureDirection: 'horizontal',
+    gestureEnabled: true,
+    headerShown: false,
+    cardStyleInterpolator: CardStyleInterpolators.forHorizontalIOS,
+};
```

Comment:
Move this to a controller file like `useNavigationController.ts`

Reply:
resolved in commit https://github.com/IagoWag/rickandmorty/commit/a71461893b6fea906309d9e3507e0c49e0d139f5